### PR TITLE
add meta label and more doc files

### PR DESCRIPTION
**Pull request by Fishrock123**
cc @phillipj

### CODE REVIEW

File: test/node-labels.test.js [hunk 1]
```diff
@@ -147,3 +147,60 @@ tap.test('label: "lib / src" when more than 5 sub-systems has been changed', (t)
...
+tap.test('label: not "meta" when other top-level have been changed', (t) => {
+  const labels = nodeLabels.resolveLabels([
+    'BUILDING.md',
+    'README.md',
+    'COLLABORATOR_GUIDE.md',
+    'CONTRIBUTING.md',
+    'configure'
+  ])
+
+  t.same(labels.meta, undefined)
```
Comment (phillipj) [inline test/node-labels.test.js]: Hmm... As `labels` is an array `labels.meta` would always be `undefined`. Probably replace this with `Array.indexOf()` or `Array.includes()` if Travis CI has v6 available (?).

**Comment by phillipj** [comment]
Linking with #31

File: test/node-labels.test.js (hunk 1 above)
Comment (Fishrock123) [inline test/node-labels.test.js]: doh

**Comment by phillipj** [comment]
LGTM 👍

**Comment by phillipj** [comment]
@Fishrock123 deployed

**Pull request merged**